kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/ServiceDiscovery.scala

```diff
@@ -24,17 +24,16 @@ import java.util.concurrent.atomic.AtomicBoolean
 import javax.security.auth.login.Configuration
 
 import scala.collection.JavaConverters._
-import scala.util.control.NonFatal
 
 import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory}
-import org.apache.curator.framework.recipes.nodes.PersistentEphemeralNode
+import org.apache.curator.framework.recipes.nodes.PersistentNode
```

Comment:
@MyLanPangzi This is not just remove unused import but also remove some warn, please note the change at the title and description.